cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -111,15 +111,15 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
         // * Channel switcher hint should be visible
         cy.get('#quickSwitchHint').should('be.visible').should('contain', 'Type to find a channel. Use ↑↓ to browse, ↵ to select, ESC to dismiss.');
 
-        // 4. Type "nt" on Channel switcher input
-        cy.get('#quickSwitchInput').type('nt');
+        // 4. Type "nesciunt" on Channel switcher input
+        cy.get('#quickSwitchInput').type('nesciunt');
         cy.wait(500);  // eslint-disable-line
 
         // * Suggestion list should be visible
         cy.get('#suggestionList').should('be.visible');
 
         // 5. Press down arrow and then enter
```

Comment:
Very minor: do you want to update the comment to remove `down arrow and then`?

Reply:
I missed that, thanks!